docs/docs/code/nodejs/ai-code-generation/README.md

```diff
@@ -1,53 +1,48 @@
 # Using AI to generate code
-![Introducing Pi](https://res.cloudinary.com/pipedreamin/image/upload/c_scale,w_425/v1685134013/image_5_yppihx.png)
```

Comment:
Removing the Pi icon for now since we actually don't use it anywhere. Pi should probably just be the face of Q&A when we add that in the product.